.github/workflows/daily_common.yml

```diff
@@ -86,4 +91,7 @@ jobs:
         run: |
           nim --version
           nimble --version
-          NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }}" nimble test
+          [[ ${{ inputs.use_sat_solver }} ]] && dependency_solver="sat" || dependency_solver="legacy"
+          NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }} --solver:$dependency_solver" 
```

Comment:
```suggestion
NIMFLAGS="--mm:${{ matrix.nim.memory_management }} --solver:${dependency_solver}"
```

Reply:
This is written as it was before: Instead of assuming `NIMFLAGS` is empty we just append the values to it, as it's common practice.

Reply:
Right, I updated it.

Reply:
It's about the missing `{}`

Reply:
It's probably better to change the name to upper case too.

Reply:
Oh, I see, I misread it.

Reply:
Ah, I see, I misunderstood.
Right, it would work regardless as `$variable` can be used interchangeably with `${variable}`; not the other way around, though.

Reply:
I'd kinda keep the lowercase, though, as uppercase is used for env variables while lowercase for local ones. IIRC.